packages/core-ethereum/src/commitment.ts

```diff
@@ -3,7 +3,7 @@ import { iterateHash, recoverIteratedHash, u8aConcat, Intermediate } from '@hopr
 import type { LevelUp } from 'levelup'
 import { randomBytes } from 'crypto'
 import Debug from 'debug'
-import { getFromDB } from './dbKeys'
+import { getFromDB } from './utils'
```

Comment:
eww, utils, can we call it 'db'

Reply:
@peterbraden it's only one function, do we need a file for it? we can create `db` once we improve DB in general